lib/collections.js

```diff
@@ -11,12 +11,23 @@ var paragraph =
 // Question 1
 // write code to split the above paragraph into an array of words
 // then normalize the words in the array
+var words = [];
+words = paragraph.split(/\s+/);
+
+//console.log(words)
 
 // The word list normalized (all lower case with punctuation removed)
-var normalizedWords = [];
 
-/* your code starts here */
 
+/* your code starts here */
+var normalizedWords = [];
+var i;
+for (i = 0; i < words.length; i++) {
+  var word = words[i];
+  word = word.replace(/[^\w']+/, '').toLowerCase();
+  normalizedWords[i] = word;
+}
```

Comment:
The variable `word` there could have been left out if you'd written
```javascript
 normalizedWords[i] = words[i].replace(/[^\w']+/, '').toLowerCase();
```
(or, alternatively, 
```javascript
 normalizedWords.push(words[i].replace(/[^\w']+/, '').toLowerCase());
```
)